src/Compiler/Microsoft.CodeAnalysis.Razor/test/DefaultTagHelperDescriptorFactoryTest.cs

```diff
@@ -1845,7 +1845,6 @@ public class DynamicTestTagHelper : {{typeof(AspNetCore.Razor.TagHelpers.TagHelp
     public void CreateDescriptor_BuildsDescriptorsFromSimpleTypes()
     {
         // Arrange
-        var objectAssemblyName = typeof(Enumerable).GetTypeInfo().Assembly.GetName().Name;
         var factory = new DefaultTagHelperDescriptorFactory(Compilation, includeDocumentation: false, excludeHidden: false);
```

Comment:
I'm really surprised that wasn't generating a build warning for unused local. Is that being suppressed here?

Reply:
Not suppressed, simply looks like we do not have a warning for "IDE0059: Remove unnecessary value assignment" enabled.

Reply:
https://github.com/dotnet/razor/pull/10428